packages/ui-components/src/components/calendar-advanced-date-selector/calendar-advanced-date-selector.types.ts

```diff
@@ -33,9 +33,16 @@ export interface ICalendarAdvanceAbsoluteTimeConfig {
 	/** (optional) Absolute time selector maximum accepted date */
 	maxDate?: string;
 	/** (optional) The start date text field input custom configurations */
-	startInputConfig?: Partial<ITextField>;
+	startInputConfig?: IInputConfig;
 	/** (optional) The end date text field input custom configurations */
-	endInputConfig?: Partial<ITextField>;
+	endInputConfig?: IInputConfig;
+}
+
+export interface IInputConfig {
```

Comment:
I think what you want here is to extend the `ITextField`, right?
```suggestion
export interface IInputConfig extends Partial<ITextField> {
```

Reply:
Yeap, Miguel was telling me that and I totally forgot mybad.